### ISSUE TITLE
[question] IATI data updated field logic

### ISSUE THREAD

**Issue by stephanoshadjipetrou** [IATI/ckanext-iati#393]
Hi

I want to ask how is the `IATI data updated` field for each dataset calculated and on which action does it get updated?

Thanks

**Comment by jodiegardiner** [comment]
From searching the code for data updated, looks like it's here:
https://github.com/IATI/ckanext-iati/blob/01f9da10e567c1db656195e0614552d05f234211/ckanext/iati/archiver.py#L288

**Comment by stephanoshadjipetrou** [comment]
Thanks @jodiegardiner!
So from what I get from this is that it is calculated based on the `last-updated-datetime` attributes from all the activities in the dataset, right?

**Comment by jodiegardiner** [comment]
Yeah it looks like it parses the xml tree of the resource and stops when it finds a date and uses that.

**Comment by robredpath** [comment]
#429 will make this clearer for users.
